andela/blackninjas-frontend
# #169817840 Users should be able to book an accommodation facility

#### What does this PR do?
- Users should be able to book an accommodation facility
- Users should be able to provide feedback on accommodation
- Users should be able to like/unlike an accommodation facility
- Users should be able to rate a centre

#### Description of Task to be completed?
- Users of Barefoot Nomad should be able to book accommodation facilities on the platform depending on what the "supplier(s)/host(s)" have to offer.

- Users should be able to rate accommodation facilities in the centres that they have travelled to.

- Users should be able to comment/provide feedback about an accommodation facility.

#### How should this be manually tested?
After clone repository, go to the root of the project open path into command run the `npm install` and once setup is done click `npm start-dev`, this will start a local server which will then automatically open your default browser, then navigate to `http://localhost:8080/trips` and the click on a given trip to see full details and from there if your trip request is approved then you will see `Book accommodation button` which will take you to a booking page and there you will see all info of an accommodation you want to book from the proceed with the booking process.

#### What are the relevant pivotal tracker stories?
- [#169817840](https://www.pivotaltracker.com/story/show/169817840)
- [#169817841](https://www.pivotaltracker.com/story/show/169817841)
- [#169817843](https://www.pivotaltracker.com/story/show/169817843)
- [#169817842](https://www.pivotaltracker.com/story/show/169817842)

#### Screenshots (if appropriate)

<img width="1440" alt="Screen Shot 2020-03-24 at 21 12 47" src="https://user-images.githubusercontent.com/28502544/77468008-9fa54b00-6e15-11ea-9894-dc4444c142a0.png">

<img width="1440" alt="Screen Shot 2020-03-24 at 21 15 00" src="https://user-images.githubusercontent.com/28502544/77468247-fca10100-6e15-11ea-819c-7aef935db45d.png">

<img width="1440" alt="Screen Shot 2020-03-24 at 21 14 52" src="https://user-images.githubusercontent.com/28502544/77468080-c2376400-6e15-11ea-890a-c6ed53f73b6a.png">

<img width="376" alt="Screen Shot 2020-03-24 at 21 13 10" src="https://user-images.githubusercontent.com/28502544/77468402-28bc8200-6e16-11ea-877d-7d851e8d5122.png">

<img width="359" alt="Screen Shot 2020-03-24 at 21 13 55" src="https://user-images.githubusercontent.com/28502544/77468414-2d813600-6e16-11ea-9db2-9c7f622a57ed.png">

<img width="374" alt="Screen Shot 2020-03-24 at 21 14 31" src="https://user-images.githubusercontent.com/28502544/77468460-3b36bb80-6e16-11ea-8d7b-d79c1833f281.png">
